addons/mass_mailing/models/mail_mail.py

```diff
@@ -44,6 +44,21 @@ def _get_unsubscribe_url(self, email_to):
         )
         return url
 
+    def _get_browser_view_url(self, email_to):
```

Comment:
If we go for a route like "/mailing/<int>/view", we could update its naming to _get_view_url (to be coherent, I let you choose what do to :) )